src/playbacks/hls/hls.js

```diff
@@ -124,6 +123,12 @@ export default class HLS extends HTML5VideoPlayback {
     }
   }
 
+  render() {
+    var a = super.render()
```

Comment:
:) I think you're right! maybe we can get rid of that `a`, my observation was about naming that `a` could mislead or make us to go to `HTML5Video` to understand what's `a`. I think, `return this` is best.

```javascript
render() {
  super.render()
  this.ready()
  return this
}
```